webapi/src/main/java/com/dumbdogdiner/stickyapi/webapi/mcapi/Ping.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.webapi.mcapi;
+
+class Ping {
```

Comment:
one class for just two vars? please move these to wherever they are used, thanks!

Reply:
Very much agree with koda here - what's this used for? Likely for ping status yes, but imo this is slightly overkill.